src/styles/utils/generators/TooltipStyleUtils/index.native.ts

```diff
@@ -0,0 +1,179 @@
+import {Animated, StyleSheet} from 'react-native';
+import FontUtils from '@styles/utils/FontUtils';
+// eslint-disable-next-line no-restricted-imports
+import type StyleUtilGenerator from '@styles/utils/generators/types';
+// eslint-disable-next-line no-restricted-imports
+import positioning from '@styles/utils/positioning';
+// eslint-disable-next-line no-restricted-imports
+import spacing from '@styles/utils/spacing';
```

Comment:
Why are we restricting these imports but using them anyways?

Reply:
Why can't we use hooks?